pastis/launchers/run_luvex_efields.py

```diff
@@ -48,13 +49,21 @@
     # Calculate the static tolerances
     c_target = 6.3*1e-11
     mus = calculate_segment_constraints(pastis_matrix, c_target=c_target, coronagraph_floor=contrast_floor)
-    np.savetxt(os.path.join(dir_run, 'mus_%.2e_%d.csv' % (c_target, NUM_RINGS)), mus, delimiter=',')
+    np.savetxt(os.path.join(dir_run, f'mus_{c_target:.2e}_{NUM_RINGS:d}.csv'), mus, delimiter=',')
 
-    num_modes = 10 # for harris thermal map or number of localized zernike modes = "DM_SPEC"
+    num_modes = 5   # for Harris thermal map or number of localized zernike modes = "DM_SPEC"
     nseg = run_matrix.simulator.nseg
 
-    ppl.plot_zernike_mus(mus, num_modes, nseg, c_target, dir_run, save=True)
-    # ppl.plot_thermal_mus(mus, num_modes, nseg, c_target, dir_run, save=True)
+    coeffs_table = util.sort_1d_mus_per_segment(mus, num_modes, nseg)
+    mu_list = []
+    label_list = []
+    for i in range(coeffs_table.shape[0]):
+        mu_list.append(coeffs_table[i])
+        if WHICH_DM == 'seg_mirror':
+            label_list.append(f'Zernike mode {i}')
+    if WHICH_DM == 'harris_seg_mirror':
+        label_list = ['Faceplates Silvered', 'Bulk', 'Gradient Radial', 'Gradient X lateral', 'Gradient Z axial']
```

Comment:
I feel this contradicts your statement regarding keeping launcher scripts thinner and lighter. I was expecting you to do calculation somewhere else. 
"_As for the part in the launcher, I would suggest to actually get started on a new analysis framework, rather than putting things into the launcher._" in the comment section of my PR #141